65.md

```diff
@@ -8,7 +8,17 @@ Relay List Metadata
 
 Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. Relays marked as `read` / `write` are called READ / WRITE relays, respectively. If the marker is omitted, the relay is used for both purposes.
+The event MUST include a list of `r` tags with relay URIs with an optional single marker indicating what the relay is used for.
+
+The marker, if it exists, MUST be one of:
+
+- `write` - The user writes events to this relay for the public to view.
+- `read` - The user reads events from this relay if they tag the user.
+- `dm` - This is identical to `read` but also signals that encrypted direct messages should preferably be written or one of these relays.
```

Comment:
The last change in this PR suggests that DM is both a READ and a WRITE relay: 

> DMs SHOULD only be broadcasted to the recipient's and author's DM relays.

Which I think it is correct, but here it says that it only works as a READ relay, which I think it is incorrect. DM relays are indeed for both write and read. 

PS: The READ base works for the current GiftWrap design (the signing key is always random and thus no WRITE is needed), but there may be other encryptions that are signed by the author.

Reply:
Is there a case where somebody comes to your DM relay to read your DMs?  If not, it is inbox only. Just because you write into it doesn't make it an outbox.  I would leave it more flexible, except that if we make these outboxes, what was the point of special DM relays that leak your DMs?

Reply:
> Is there a case where somebody comes to your DM relay to read your DMs?

Only for those gathering metadata from DMs (which does happen today already). Do we consider a shared-secret group, like what @staab has, a "DM"? They are encrypted messages but anyone with the key can access them.  

It wasn't clear to me that this PR requires DM relays to authenticate who is connecting and only provide the appropriate messages to that user. If that is the case then it makes sense to be read. Maybe this is an expected behavior of DM relays and should be added here?

Reply:
Seems to me that it should act like a read relay. Auth should be recommended but optional. This works for shared key groups too, but isn't relevant to nip 65 because the group metadata defines the relays to use, not the recipient's profile.

Reply:
I think it depends on the specific DM spec. I was imaging a copy lands in my read relay (for me) and my partner's read relay (for them). But maybe this NIP should defer to the DM NIPs.